backend/lib/config/gardener.js

```diff
@@ -126,6 +126,11 @@ module.exports = {
       _.unset(config, 'frontend.ticket')
     }
 
+    if (config.frontend.shootAdminKubeconfig) {
+      _.set(config, 'frontend.shootAdminKubeconfig.enabled', config.frontend.shootAdminKubeconfig.enabled)
+      _.set(config, 'frontend.shootAdminKubeconfig.maxExpirationSeconds', config.frontend.shootAdminKubeconfig.maxExpirationSeconds)
+    }
+
```

Comment:
Why do we need this?

Reply:
I thought this was needed for /api/config to serve the shootAdminKubeconfig settings. I removed it.